src/sink/wavefront.rs

```diff
@@ -215,6 +308,7 @@ impl Sink for Wavefront {
     }
 
     fn flush(&mut self) {
+        self.format_stats(time::now());
```

Comment:
I believe you're correct, that said, it should be `Instant::now` to have the same monotonicity.